src/Containers.jsx

```diff
@@ -40,19 +41,126 @@ class Containers extends React.Component {
         }));
     }
 
-    //TODO
-    stopContainer(container) {
-        return undefined;
+    stopContainer(container, timeout) {
+        document.body.classList.add('busy-cursor');
+        const id = container.ID;
+        if (!timeout) {
+            timeout = 10;
+        }
+        utils.varlinkCall(utils.PODMAN, "io.podman.StopContainer", JSON.parse('{"name":"' + id + '","timeout":' + timeout + '}' ))
```

Comment:
Don't go through efforts to construct a JSON string and then immediately parse it back. This is error prone (e. g. if a container name contains `'`, and hard to read. Just give a JS object, i. e. the argument is `{name: id, timeout}`. This applies to the other calls as well, of course.